### PR TITLE
Tjackson2

### PR DESCRIPTION
**Pull request by AndersonTylerJackson**
Hello, world.(?) Adding my magic test.

### CODE REVIEW

**Comment by ktarplee** [comment]
Remove the non .py files such as .pyc and .idea from the git repo.

File: tjackson2/lab4_test.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'''09/28/2015 Tyler Jackson Testing Implementation for magic tests.
+    Range of all whole numbers from 100 through 999.'''
+from . import lab4
+
+
+def magic_or_not_test():
+    for i in range(100, 1000):
+        if lab4.decreasing_or_not(i) is True:
```
Comment (ktarplee) [inline tjackson2/lab4_test.py]: remove "is True" as it is redundant (mostly).

File: tjackson2/lab4_test.py~ [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'''09/28/2015 Tyler Jackson Testing Implementation for magic tests.
```
Comment (ktarplee) [inline tjackson2/lab4_test.py~]: Delete this tilde file you just added.  This is a backup file or emacs (often).

File: tjackson2/lab4.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+def decreasing_or_not(pInt):
+    int_array = list(map(int, str(pInt)))
+    for i in range(0, len(int_array)-1):
+        if int_array[i] <= int_array[i+1]:
+            return False
+    return True
+
+
+def reverse_number(pInt):
+    int_string = str(pInt)
```
Comment (ktarplee) [inline tjackson2/lab4.py:19]: add docstrings

File: tjackson2/lab4.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+'''09/28/2015 Tyler Jackson  Functions that handle testing for magic numbers.
+    Includes a function to test to see if the number
+    is in magic number format, meaning decreasing,
+    and a function to reverse a number.'''
+
+
+def decreasing_or_not(pInt):
+    int_array = list(map(int, str(pInt)))
```
Comment (ktarplee) [inline tjackson2/lab4.py:10]: add docstrings

File: tjackson2/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+__author__ = 'user1'
```
Comment (ktarplee) [inline tjackson2/__init__.py]: Can you remove this line (user1 is not very informative).

File: tjackson2/lab4_test.py~ (hunk 2 above)
Comment (AndersonTylerJackson) [inline tjackson2/lab4_test.py~]: The tilde files aren't showing up in the file explorer, I'm not sure where they keep coming from or how to delete them.

**Comment by ktarplee** [comment]
I usually add *~ to my global gitignore file so I never commit them.

**Pull request merged**